LayoutTests/svg/crash-svg-filter-empty-viewport.svg

```diff
@@ -0,0 +1,27 @@
+<svg width="0" height="0" xmlns="http://www.w3.org/2000/svg" onload="test()">
+  <script href="../resources/js-test.js"/>
+  <filter id="filter"></filter>
+  <marker id="marker" orient="1" filter="url(#filter)"></marker>
+  <line marker-end="url(#marker)"></line>
+  <text>PASS</text>
+  <script>
+    function test() {
+      let filter = document.getElementById("filter");
+      x = filter.x.baseVal;
+      shouldBe('x.unitType', "SVGLength.SVG_LENGTHTYPE_PERCENTAGE");
```

Comment:
Please add a `description('...');` statement here to describe what this test is doing.

Reply:
Done.